backend/src/API/siteIntegrationAPI.ts

```diff
@@ -19,12 +25,24 @@ export const requestIDOLPullDispatch = async (user: IdolMember): Promise<{ updat
   return { updated: result.status === 204 };
 };
 
+/**
+ * Retrieves the latest Pull Request with changes from the IDOL backend.
+ * @param {IdolMember} user - The user requesting the pull request details.
+ * @returns {Promise<{pr: PRResponse}>} - A promise that resolves to an object containing the pull request details.
+ * @throws {PermissionError} - If the user lacks permission to access the pull request.
+ */
 export const getIDOLChangesPR = async (user: IdolMember): Promise<{ pr: PRResponse }> => {
   await checkPermissions(user);
   const foundPR = await findBotPR();
   return { pr: foundPR };
 };
 
+/**
+ * Accepts and merges the IDOL backend changes pull request.
```

Comment:
This should mention what happens when the PR is merged (i.e. triggering a deploy).